github/handlers/HandleInvalidRepoName.ts

```diff
@@ -0,0 +1,56 @@
+import {
+    IHttp,
+    IModify,
+    IRead,
+} from "@rocket.chat/apps-engine/definition/accessors";
+import { GithubApp } from "../GithubApp";
+import { isRepositoryExist } from "../helpers/githubSDK";
+import { getAccessTokenForUser } from "../persistance/auth";
+import { IUser } from "@rocket.chat/apps-engine/definition/users";
+import { IRoom } from "@rocket.chat/apps-engine/definition/rooms";
+import { IAuthData } from "@rocket.chat/apps-engine/definition/oauth2/IOAuth2";
+
+export async function HandleInvalidRepoName(
+    repoName: string,
+    http: IHttp,
+    app: GithubApp,
+    modify: IModify,
+    sender: IUser,
+    read: IRead,
+    room: IRoom
+): Promise<boolean> {
+    const accessTokenInfo = (await getAccessTokenForUser(
+        read,
+        sender,
+        app.oauth2Config
+    )) as IAuthData;
+
+    const access_token = (accessTokenInfo == undefined) ? undefined : accessTokenInfo.token;
+    
+    const isValidRepository: boolean = await isRepositoryExist(
+        http,
+        repoName,
+        access_token
+    );
+
+    if (!isValidRepository) {
+        const warningBuilder = await modify
+            .getCreator()
+            .startMessage()
+            .setRoom(room)
+            .setText(
+                `Hey ${sender.username}! Provided repository doesn't exist or you need to login to access private repo.`
+            );
```

Comment:
@Nabhag8848 I think this text should be `Hey ${sender.username}! Provided repository doesn't exist` if the access token does exist. If the access token does not exist the text can be `Hey ${sender.username}! Provided repository doesn't exist or you need to login to access private repo.`.  Conditionally rendering something like this, will be low effort but lead to a better user experience.